src/worker.cc

```diff
@@ -685,6 +720,43 @@ void from_json(rapidjson::Document& doc, Options& options) {
     options.set_linear_references(*linear_references);
   }
 
+  // costing defaults to none which is only valid for locate
```

Comment:
This code block was moved up before parsing locations so that we have the costing options parsed before hand. This helps us access `ignore_closures` option later.